pkg/virtctl/ssh/knownhosts_test.go

```diff
@@ -0,0 +1,52 @@
+package ssh
+
+import (
+	"bufio"
+	"crypto/rand"
+	"crypto/rsa"
+	"os"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"golang.org/x/crypto/ssh"
+)
+
+var _ = Describe("Known Hosts", func() {
+
+	var tmpDir string
+	BeforeEach(func() {
+		tmpDir = GinkgoT().TempDir()
+	})
+
+	It("should be added with a newline", func() {
+		knowHostFile := filepath.Join(tmpDir, "knownhosts")
```

Comment:
maybe inline both `GinkgoT().TempDir()` here ?
both have automatic DeferCleanup with ginkgo 2.0

Reply:
I can move the TempDir to the test body.